src/app/splash/splash.component.html

```diff
@@ -1,23 +1,38 @@
-<div class="container-background" (click)="navigateToWelcome()" (keypress)="navigateToWelcome()" tabindex="0">
-  <button class="logo">
-    <img src="assets/logo/IO-logo-inverted_transparent-bg.png" alt="logo" />
-  </button>
-
-  <app-speech-bubble
-    class="speech-bubble"
-    [textColor]="textColor"
-    [bodyText]="'Slik ser et eple ut, O!'"
-    [pointerSide]="PointerSide.Bottom"
-    [arrowAlignment]="ArrowAlignment.Left"
-    [bubbleSize]="'large'"
-  >
-  </app-speech-bubble>
-
-  <img class="teaching-image" src="assets/I-teaching-O.svg" alt="I teaching O" />
-
-  <div class="play-text">Trykk for å spille</div>
-</div>
-
-<!-- Bør heller ta inn translation pipe:
-[bodyText]="'SPEECH_BUBBLE_TEXT' | translate"
-<div class="play-text">{{ 'PRESS_TO_START' | translate }}</div> -->
+<div class="container-background" (click)="navigateToWelcome()" (keypress)="navigateToWelcome()" tabindex="0">
+    <button class="logo">
+        <img src="assets/logo/IO-logo-inverted_transparent-bg.png" alt="logo" />
+    </button>
+    
+    <div class="speech-bubble-container">
+        <app-speech-bubble
+            *ngIf="isFirstSpeechBubble"
```

Comment:
A suggestion here: 
Maybe use the @if(){..} annotation instead of the old *ngIf, so it matches the rest of the code base? 🤔

Reply:
I very much agree!

Edited, but it messes up the styling. I will fix this after the demo!